sap_hana/README.md

```diff
@@ -0,0 +1,111 @@
+# Agent Check: SAP HANA
+
+## Overview
+
+This check monitors [SAP HANA][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The SAP HANA check is included in the [Datadog Agent][2] package.
+
+#### Prepare HANA
+
+To query certain views, specific privileges must be granted to the chosen HANA monitoring user.
+
+##### User creation
+
+1. Connect to the system database and run the following command to create a user:
+
+```
+CREATE RESTRICTED USER <USER> PASSWORD <PASSWORD>
+```
+
+2. Run the following command to allow the user to connect to the system:
+
+```
+ALTER USER <USER> ENABLE CLIENT CONNECT
+```
+
+3. (optional) To avoid service interruption you may want to make the password long-lived:
+
+```
+ALTER USER <USER> DISABLE PASSWORD LIFETIME
+```
+
+##### Granting privileges
+
+1. Run the following command to create a monitoring role (we'll call it `DD_MONITOR` for these examples):
+
+```
+CREATE ROLE DD_MONITOR
+```
+
+2. Run the following command to grant read-only access to all system views:
+
+```
+GRANT CATALOG READ TO DD_MONITOR
+```
+
+3. Then run the following commands:
```

Comment:
Then run the following commands to do what? All of the other steps describe what you're doing and this one should too.